docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -65,6 +65,15 @@ The complete list of available targets is the following:
         <td><code>jvm</code></td>
         <td></td>
     </tr>
+    <tr>
+        <td rowspan="2">Kotlin/Wasm</td>
+        <td><code>wasmJs</code></td>
+        <td>Use it for <a href="wasm-js-interop.md">interoperability with JavaScript</a>.</td>
```

Comment:
It is more not about interoperability but about runtimes with JS support
Something like "if you plan to run in JavaScript runtime"